tests/utils/deleteNearbyEntities.js

```diff
@@ -1,6 +1,8 @@
-// Remove all entities near the avatar
+// Remove all entities near (i.e. with 1000m from) the avatar
 const TEST_RADIUS = 1000;
 var entitiesToDelete = Entities.findEntities(MyAvatar.position, TEST_RADIUS);
 for (var i = 0; i < entitiesToDelete.length; ++i) {
     Entities.deleteEntity(entitiesToDelete[i]);
 }
+
+Script.stop();
```

Comment:
Is this equivalent to wrapping everything in
```
(function() {
...
}());
```

Reply:
I tried that - this doesn't stop the script.  I need the script to stop so that Interface will shutdown, and testing can proceed.

Reply:
huh ok